server/emails/ExportSucessEmail.js

```diff
@@ -8,25 +8,30 @@ import Footer from "./components/Footer";
 import Header from "./components/Header";
 import Heading from "./components/Heading";
 
-export const exportEmailText = `
+export const exportEmailSuccessText = `
 Your Data Export
 
-Your requested data export is attached as a zip file to this email.
+Your requested data export is complete, the exported files are also available in the admin section.
 `;
 
-export const ExportEmail = () => {
+export const ExportSuccessEmail = ({ id }: { id: string }) => {
   return (
     <EmailTemplate>
       <Header />
 
       <Body>
         <Heading>Your Data Export</Heading>
         <p>
-          Your requested data export is attached as a zip file to this email.
+          Your requested data export is complete, the exported files are also
+          available in the admin section.
         </p>
         <EmptySpace height={10} />
         <p>
-          <Button href={`${process.env.URL}/home`}>Go to dashboard</Button>
+          <Button
```

Comment:
We need to pass `teamUrl` and use it here (same as welcome email)  instead of the `process.env.URL` otherwise on cloud hosted it will send them to `https://app.getoutline.com/api/... where they won't be signed in.